boudicca.base/fetcher-lib/src/main/kotlin/base/boudicca/fetcher/Fetcher.kt

```diff
@@ -10,9 +10,13 @@ import java.net.http.HttpResponse.BodyHandlers
 import java.time.Clock
 import java.util.concurrent.Callable
 
+private const val WAIT_TIME_FACTOR = 1.5F
+private const val MIN_WAIT_TIME = 100L
+
 /**
  * THIS CLASS IS NOT THREADSAFE!
  */
+@Suppress("detekt:LongParameterList")
```

Comment:
instead of supressing, why no create a data class called FetcherConfig?

not sure if detekt also complains about data classes, but maybe this would be a bit cleaner anyway?

Reply:
not sure if that makes it better? i think it is ok this way, especially because all those parameters have default values anyway